GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -2755,6 +2764,14 @@ private void Message_KeyDown(object sender, KeyEventArgs e)
                 ExecuteCommitCommand();
                 e.Handled = true;
             }
+
+            if (e.Control && e.KeyCode == Keys.T)
```

Comment:
I can imagine users wanting shortcuts for some of the others here too.

How about using Ctrl + Shift, and also supporting e.g. `x` for `fix`, and others?

Reply:
ctrl+T is more to easily open the menu. After, it's easy enough (IMHO) to select the good one with arrows or the cycle using the first char of the keyword.

But you gave me an idea on a good way to use the Ctrl + Shift +T to add also "()" for the "scope". I tried with "ctr/shift/alt+enter" when selecting and none is working 😢

Reply:
Perhaps, hotkeys work better, which is recommended anyway.

Reply:
I will have a look for hotkeys without guarantee (since I added the support of scopes with Ctrl + Shift +T)

Reply:
Does the Ctrl+Shift+T appear anywhere in the UI? How do we expect users to know about this feature?

Reply:
I have switch to use hotkey manager so it will be visible in the settings.

Reply:
And I have displayed both hotkeys in the menu (screenshot updated in PR description)